packages/valory/skills/abstract_round_abci/behaviour_utils.py

```diff
@@ -372,9 +373,11 @@ def __init__(self, **kwargs: Any):
         # If `ipfs_domain_name` is not specified for the skill, then we get a `None` default.
         # Therefore, `IPFSBehaviour` will be disabled.
         domain = getattr(self.params, "ipfs_domain_name", None)  # type: ignore  # pylint: disable=E1101
+        loader_cls = kwargs.pop("loader_cls", Loader)
+        storer_cls = kwargs.pop("storer_cls", Storer)
```

Comment:
What's this for?

Reply:
Now we have a basic `Loader` and `Storer` on the abstract skill, which can only load and store `json` files. However, we can create any store or loader, and pass it here. This is how we pass the extended versions for the APY skill, which also support csvs and pipelines.